### ISSUE TITLE
[Mobile Layout][Dropdown] Users cannot read the intact text if text overflowed in Dropdown Callout Item

### ISSUE THREAD

**Issue by neil-ji** [microsoft/fluentui#16364]
### Environment Information

- **Package version(s)**: 7.77.0
- **Browser and OS versions**: any version

### Please provide a reproduction of the bug in a codepen:
Codepen URL: [https://codepen.io/liubaix/pen/ExgLNYV?editors=0010](https://codepen.io/liubaix/pen/ExgLNYV?editors=0010)
Reproduce it by browser devtool ( device toolbar ). 
Mobile users has no mouse event therefore Tooltip cannot working, how to make sure they can read the intact text?

#### Actual behavior:
The text is truncated and displayed as an ellipsis therefore users cannot read it integrally.

#### Expected behavior:
Display text in a new line or  pop-up ( maybe other better design ).

### Priorities and help requested:

Are you willing to submit a PR to fix? (No)

Requested priority: (High)

**Comment by neil-ji** [comment]
Hi, @layershifter . Do you have any plans to deal with this proble? Any feedback may help us solve this problem, thx.

**Comment by neil-ji** [comment]
![dropdown_bug_1](https://user-images.githubusercontent.com/45644652/105787747-11ac6a80-5fba-11eb-8d26-7e70cf1d979c.png)
We think this topic is important for mobile user, do you have some better proposals or answers? In addition, if this topic is not the key point, please let me know, and I will try another ways.

**Comment by gouttierre** [comment]
@neil-ji  - Thanks for your patience on this topic. If you have a proposal on the implementations, you can submit a feature to this by submitting a pull request to this github. This [section ](https://github.com/microsoft/fluentui/blob/master/packages/react/README.md#building-the-repo) of the readme provides some good getting started information! Additionally we are certainly here to help.

@khmakoto  - Would you be able to determine if there is possible work around that would exposed the ellipse text (inline) on a mobile platform? If not, we can add this request to the backlog as a feature request. 
cc @smhigley

**Comment by khmakoto** [comment]
@gouttierre for sure, there is a work around for this that can expose this for mobile if needed via some styling overrides. an example of that could be found [in this codepen](https://codepen.io/khmakoto/pen/abBPxjj?editors=0010).

@neil-ji let us know if that helps!

**Comment by neil-ji** [comment]
@layershifter  @khmakoto Sorry for delay.

We have closed this issue after discussion, it is a small probability event ( the most text is short and simple ), we might offer a new interaction if customers definitely ask for solving it.

By the way, I have tried to change the styling, just like your offered demo, it can solve the issue but not the best way, because it need to change the styling of every different status( such as selected/hover/active/etc. ), we have to also consider the side effect on the original styling, the key point is it make the views looked very discordant on the mobile.

I'm looking up the best practice of this issue, if I find it I might submit a PR or open an another issue, so I guess you can close this one. whatever, thanks for your effort and have a good day. :smile:

**Comment by khmakoto** [comment]
@neil-ji sounds good! We'd welcome a PR or another issue if you find anything regarding this! I'll close this issue for now then!

Thanks!
